__tests__/app.test.js

```diff
@@ -12,35 +12,31 @@ beforeEach(() => {
   return seed(testdata);
 });
 
-xdescribe("GET /api/topics", () => {
+describe("GET /api/topics", () => {
   test("GET respond with an array of topics", () => {
     return request(app)
       .get("/api/topics")
       .expect(200)
       .then((res) => {
-        // console.log(res.body);
         expect(res.body.topics.length).toBe(3);
       });
   });
 });
 
-xdescribe("GET /api/articles/:article_id", () => {
-  test("GET respond with an array of articles based on id", () => {
-    return request(app)
-      .get("/api/articles/5")
-      .expect(200)
-      .then((res) => {
-        // console.log(res.body.articles);
-        expect(res.body.article).toEqual({
-          article_id: expect.any(Number),
-          title: expect.any(String),
-          topic: expect.any(String),
-          author: expect.any(String),
-          body: expect.any(String),
-          created_at: expect.any(String),
-          votes: expect.any(Number),
-        });
-      });
+describe("GET /api/articles/:article_id (comment count)", () => {
+  test("GET respond with an array of articles based on id", async () => {
+    const res = await request(app).get("/api/articles/2").expect(200);
+
+    expect(res.body.article).toEqual({
+      article_id: expect.any(Number),
+      title: expect.any(String),
```

Comment:
`expect.any()` is great in lots of cases, but here we want the `article_id` to specifically be `2` since that is the one you're requesting!